src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/PortableInfoboxPageObject.java

```diff
@@ -62,36 +59,40 @@
   @FindBy(css = "#articleCategories .category a")
   private List<WebElement> categories;
   @FindBy(css = ".pi-navigation a[href*='redlink']")
-  private List<WebElement> pInfoRedlLinkList;
+  private List<WebElement> redlLink;
+  @FindBy(css = ".pi-item .external")
+  private List<WebElement> externalLink;
   @FindBy(css = "b")
   private List<WebElement> boldElements;
   @FindBy(css = "i")
   private List<WebElement> italicElements;
   @FindBy(css = ".portable-infobox a[href*='/wiki/']")
-  private List<WebElement> pInfoInternalLinkList;
+  private List<WebElement> internalLink;
   @FindBy(css = ".pi-data-value ul li")
-  private List<WebElement> unorderedElementList;
+  private List<WebElement> unorderedElement;
   @FindBy(css = ".pi-data-value ol li")
-  private List<WebElement> orderedElementList;
+  private List<WebElement> orderedElement;
   @FindBy(css = ".pi-header")
-  private List<WebElement> pInfoTitleH3;
+  private List<WebElement> titleH3;
 
   public PortableInfoboxPageObject(WebDriver driver) {
     super(driver);
   }
 
   public String getBackgroundColor() {
-    return pInfoLayout.getCssValue("background-color");
+    return layout.getCssValue("background-color");
   }
 
-  public String getExternalLinkRedirectTitle() {
-    wait.forElementVisible(pInfoExternalLink);
-    return pInfoExternalLink.getAttribute("href");
+  public String getExternalLinkRedirectTitleWithIndex(int index) {
+    wait.forElementVisible(externalLink.get(index));
+    scrollToElement(externalLink.get(index));
+    return externalLink.get(index).getAttribute("href");
   }
 
-  public String getInternalLinkRedirectTitle(int index) {
-    wait.forElementVisible(pInfoInternalLinkList.get(index));
-    return pInfoInternalLinkList.get(index).getAttribute("href");
+  public String getInternalLinkRedirectTitleWithIndex(int index) {
```

Comment:
The code in this and above method does the same thing, just on different link element. Have you considered extracting it so you can just call:
getLinkRedirectTitle(externalLinks.get(index))
getLinkRedirectTitle(internalLinks.get(index))